cmd/virt-handler/virt-handler.go

```diff
@@ -88,7 +89,8 @@ func main() {
 	}
 	broadcaster := record.NewBroadcaster()
 	broadcaster.StartRecordingToSink(&kubecorev1.EventSinkImpl{Interface: coreClient.Events(api.NamespaceDefault)})
-	recorder := broadcaster.NewRecorder(kubev1.EventSource{Component: "virt-handler", Host: *host})
+	// TODO what is scheme used for in Recorder?
+	recorder := broadcaster.NewRecorder(kubeapi.Scheme, kubev1.EventSource{Component: "virt-handler", Host: *host})
```

Comment:
Out of all these changes, this is the only one I didn't quite understand.  The recorder takes in the scheme as an argument now. I'm not entirely sure what I did here makes sense.